Update hardware wallets support (#902)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

::::details
### What hardware wallets does Wasabi support?

> Wasabi supports a variety hardware wallets thanks to the [Bitcoin Core Hardware Wallet Interface (HWI)](https://github.com/bitcoin-core/HWI).

Reviewer: ```suggestion
Wasabi uses the Bitcoin Core [Hardware Wallet Interface (HWI)](https://github.com/bitcoin-core/HWI) which allows it to support a variety of hardware wallets.
```

> For the complete list of all the officially supported hardware wallets by Wasabi Wallet, click [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/WasabiCompatibility.md).

Reviewer: ```suggestion
For the complete list of all the officially supported hardware wallets by Wasabi Wallet, click [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/WasabiCompatibility.md#officially-supported-hardware-wallets).
```

> For the complete list of all the officially supported hardware wallets by Wasabi Wallet, click [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/WasabiCompatibility.md#officially-supported-hardware-wallets).

Reviewer: ```suggestion
For the complete list of all the officially supported hardware wallets, click [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/WasabiCompatibility.md#officially-supported-hardware-wallets).
```
